docs/load_tests/quick_tests/2022-03-22/Load-test-2022-03-22.md

```diff
@@ -0,0 +1,109 @@
+# Load test n°7, 24/02/2022
+
+## Context
+
+We want to test the newly implemented HPA in our [jitsi-k8s](https://github.com/openfun/jitsi-k8s) infrastructure.
+
+The goals of this test is multiple:
+
+- Verify that the HPA is working
+- See how long will Scaleway take to up a node in case of high load
+- Know how jitsi will behave in case of high load
+
+---
+
```

Comment:
I did not test the downscaling of the deployment nor that of the cluster, but according to our observations, Scaleway takes ~15min to remove an "unused" node (once there is no JVB pod on the node).

The downscaling of JVB pods and nodes is very tricky to test because it will largely depend on the behaviour of users. JMT instances will always leave their meetings once the test time has elapsed.